content/about.njk

```diff
@@ -34,5 +41,60 @@ layout: layouts/base.njk
+            <h4>What makes this activity unique?</h4>
+            <p>Stuyvesant High School has a rich ecosystem of computer science/programming clubs, but no active clubs expressly emphasize and embrace Free/Libre Open Source Software. Unlike many of the school's other clubs, Stuy Linux aims for a language-agnostic approach to its content instead of focusing on a specific programming language. In addition, Stuy Linux was founded in response to a narrowly-scoped and inactive Cybersecurity club that existed in 2022, serving as an active and competitive alternative to resume-padding clubs. Stuy Linux also discusses the intricacies Unix-like systems beyond GNU/Linux, such as the BSD and its derivatives, like the XNU kernel and its commonly-known implementations in the MacOS and iOS, along with the Linux kernel's significant applications in Android, ChromeOS, and others. Our community is noteworthy for its online nature, which allows us to remain connected to interested individuals and alumni who remain interested in FOSS topics. These features were not within the scope of Stuyvesant communities that existed at the time of Stuy Linux's inception, and remain distinguishing aspects of the club to this day.</p>
+        </details>
+        <br>
+        <h2>Contact Us</h2>
+        <p>
+            Want to reach us?
+            <br/>
+            <br/>
+            Email: stuylinux[at]tutanota[dot]com
+            <br/>
+            Discord: <a href="https://discord.gg/8pkfP7mU78" target="_blank">discord.gg/8pkfP7mU78</a>
+            <br/>
+            IRC channel: <a href="https://irc.stuylinux.org" target="_blank">irc.stuylinux.org</a>
+        </p>
+        <br>
+        <h2>Subdomains</h2>
+            <hr>
+            <a href="https://among.stuylinux.org" target="_blank">among.stuylinux.org</a>
+            <p>A version of the popular game Among Us, made by a club member.</p>
+            <hr>
+            <a href="https://files.stuylinux.org" target="_blank">files.stuylinux.org</a>
+            <p>A simple nginx file server</p>
+            <hr>
+            <a href="https://git.stuylinux.org" target="_blank">git.stuylinux.org</a>
+            <p>Stuy Linux's self hosted <a href="https://about.gitea.com/" target="_blank">gitea</a> instance.</p>
+            <hr>
+            <a href="https://irc.stuylinux.org" target="_blank">irc.stuylinux.org</a>
+            <p>Stuy Linux's self hosted IRC chanel.</p>
+            <hr>
+            <p>mindustry.stuylinux.org</p>
+            <p>Stuy Linux's mindustry server.</p>
```

Comment:
(probably don't advertise this one since we only use it during meetings, same with anything Minecraft)